smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -526,14 +526,21 @@ def pollResponse(response, data) {
 		activities.each { activity ->
 			if ("${activity.deviceNetworkId}".contains("harmony-${it.key}")) {
 				//C2C-4 Temporarily disabiling device health for Logitech.
-				activity.sendEvent(name: "DeviceWatch-DeviceStatus", value: "online", displayed: false, isStateChange: true)
```

Comment:
Comment above can be removed

Reply:
I will do that :)

Reply:
https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3382